templates/web/fixmystreet.com/report/new/roads_message.html

```diff
@@ -31,12 +31,21 @@
     <div id="js-not-council-car-park" class="hidden js-responsibility-message">
         <p>There is no council car park in the area selected.</p>
     </div>
+    <div id="js-brent-road-services" class="hidden js-responsibility-message">
+        <p><strong>Public Highway issue</strong></p>
+        <p>To report this issue, please select a point on a public road</p>
+    </div>
     <div id="js-brent-tenant-services" class="hidden js-responsibility-message">
-        <p><strong>Please use Brent Council's estate services page</strong></p>
-        <p>For issues relating to housing estates in Brent, please use the Brent Council <a href="https://www.brent.gov.uk/housing/tenant-services/estate-services#estatecleaning">Estate services page</a> directly</p>
+            <p><strong>Please use Brent Council's estate services page</strong></p>
+            <p>For issues relating to housing estates in Brent, please use the Brent Council <a href="https://www.brent.gov.uk/housing/tenant-services/estate-services#estatecleaning">Estate services page</a> directly</p>
+    </div>
+    <div id="js-brent-park-services" class="hidden js-responsibility-message">
+            <p><strong>Park issue</strong></p>
+            <p>To report this issue, please select a point within a park</p>
     </div>
 </div>
```

Comment:
@dracos I am puzzled by this `div` Must be nested from outside?

Reply:
yes, you'll see it then reopens a new one which will then be closed outside the template. So this is just its way of turning what would be contents of one div into two sibling divs.